recipes/asyncify/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "asyncify" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a3/c6/33d42a1920395c9cfe912d4b6425504237b6bba47a48967aaf453b2b0d79/asyncify-0.2.2.tar.gz
+  sha256: 99cf0730feab2ee3600427336bf54dcf38ffa665436b91acf60f7c5ce6ad57a5
+
+build:
+  skip: true
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - asyncify
+  requires:
+    - pytest
+  source_files:
+    - tests
```

Comment:
As you are not running any tests you can delete the ``requires`` and ``source_files``

Reply:
aha! If I want to run pytest on my test_directory, called 'tests', what do I add to do that?

Reply:
you need to add the ``command`` section and call pytest 
Please see here more instructions
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=command#test

Reply:
Just added
```
commands:
    pytest tests
```

Reply:
Why are all my tests failling